setup.py

```diff
@@ -1,11 +1,19 @@
 from setuptools import setup, find_packages
+
 setup(
     name='nio-cli',
-    version="0.1",
-    packages=find_packages(
-        where='src'
-    ),
-    package_dir = {'': 'src'},
+
+    version='0.1.11',
+
+    description='Command line tools for n.io',
+
+    url='https://github.com/neutralio/nio-cli',
+
+    author='n.io',
+    author_email='info@n.io',
+
+    packages=['nio_cli', 'nio_cli.util', 'nio_cli.actions', 'nio_cli.nio_add_blocks'],
```

Comment:
Not sure why I had to add every package here but this is what eventually got the pypi installation to work.